### PR TITLE
Report for google analytics and tracking events

### PR DESCRIPTION
**Pull request by jvillacampa**
As the title says. Catherine, you should be able to run the report code using the login details from the mailbox.

### CODE REVIEW

File: google_analytics/google_analytics_reporting.R [hunk 1]
```diff
@@ -0,0 +1,185 @@
+# Extracting google analytics data for wider impacts
```
Comment (catherineperkins) [inline google_analytics/google_analytics_reporting.R]: Update to say HSMR public dashboard

File: google_analytics/ga_report.Rmd [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+title: "HSMR public dashboard - Google analytics report"
+author: "Public Health Scotland"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output: 
+  html_document:
+    toc: true
+    toc_float: true
+---
+  
+  ```{r TODO, include = FALSE}
```
Comment (catherineperkins) [inline google_analytics/ga_report.Rmd]: It moans about the indentation here so might be worth updating so the message doesn't appear again?

**Review by catherineperkins — APPROVED** [review]
Works well after the initial issues. Perhaps add the code to allow overwriting and maybe more explanation about that text file - unless that should be included in the SOP instead?

**Comment by catherineperkins** [comment]
Oh, and perhaps change the colours of the lines in the chart or add symbols,  I just find them a little similar. Just in case we need to share the report with anyone!

**Comment by jvillacampa** [comment]
Thanks Catherine, all good points. I have tried to deal with all of them, so I'll merge now (can't really test if the permissions thing works well though)

**Pull request merged**